### PR TITLE
Homepage remodel

### PR DESCRIPTION
**Pull request by Kameronjan**
Re modeled the homepage according to our figma design 2.0

### CODE REVIEW

**Comment by Kameronjan** [comment]
Will create footer on a new branch at a later time

**Comment by Version-LAA** [comment]
> Re modeled the homepage according to our figma design 2.0

@Kameronjan can you add a picture of what it looks like?

File: app/assets/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -13,16 +13,20 @@
...
 .mainh1 {
   font-size: 120px;
   font-weight: bold;
   font-family: 'Lato';
-  margin-top: 80px;
+  margin-top: 30px;
+  color: #FFFFFF;
 }
 
+body {
```
Comment (Version-LAA) [inline app/assets/stylesheets/pages/_home.scss:27]: @Kameronjan you can remove this from your scss. I put background color and background image styling in it's own scss file.

File: app/assets/stylesheets/pages/_home.scss [hunk 2]
```diff
@@ -13,16 +13,20 @@
...
+  background-image: image-url("wallpaper07.jpg");
```
Comment (Version-LAA) [inline app/assets/stylesheets/pages/_home.scss:28]: @Kameronjan you can remove this from your scss. I put background color and background image styling in it's own scss file.

**Review by Version-LAA — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/pages/_home.scss (hunk 2 above)
Comment (Kameronjan) [reply app/assets/stylesheets/pages/_home.scss:28]: noted!

**Comment by Kameronjan** [comment]
Screenshot 2023-06-09 at 10 03 07 PM

Just missing bacground

**Review by Version-LAA — APPROVED** [review]
nice!

**Pull request merged**